source/common/stats/stats_impl.h

```diff
@@ -103,6 +109,8 @@ struct RawStatData {
 
 private:
   static const size_t DEFAULT_MAX_NAME_SIZE = 127;
+  // total length must be equal to DEFAULT_MAX_NAME_SIZE
+  static const size_t MAX_STAT_SUFFIX_LENGTH = 67;
```

Comment:
Please only define this value in one place.  It is duplicated in options_impl.cc right now.